### DISCUSSION TITLE
Firmware 2.9.2 not support TP-Link TL-WN722-N WiFi USB?

### DISCUSSION THREAD

**Discussion by appservnetwork** [xchwarze/wifi-pineapple-cloner#25] [General]
I use with gl.inet AR300m16 
dmesg found 
usb 1-1: new high-speed USB device number 2 using ehci-platform
but I can't modprobe rtl8188
any suggestion?

**Comment by appservnetwork** [comment]
Sorry for asking.
I use TL-WN722N v1.10
This version uses the ath9k_htc chipset.

opkg update
opkg install kmod-ath9k-htc

Done !!

**Reply by wojtechit** [reply]
thank you buddy you save me a lot of time

**Comment by zeldoon** [comment]
opkg list | grep -e AR9271

ath9k-htc-firmware - 20190416-1 - AR9271/AR7010 firmware
kmod-ath9k-htc - 4.14.221+4.19.161-1-1 - This module adds support for wireless adapters based on Atheros USB AR9271 and AR7010 family of chipsets.
